### PR TITLE
Link editor tab

### PR DESCRIPTION
**Pull request by brichet**
This PR adds an empty link editor tab to the session widget:

Peek 2023-04-07 15-52

### CODE REVIEW

File: src/linkPanel/linkPanel.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { BoxPanel } from '@lumino/widgets';
+
+import { IGlueSessionSharedModel } from '../types';
+
+/**
+ * The link editor widget.
+ */
+export class LinkWidget extends BoxPanel {
+  constructor(options: LinkWidget.IOptions) {
+    super({ ...options, direction: 'left-to-right' });
```
Comment (trungleduc) [inline src/linkPanel/linkPanel.ts:11]: I think we could take the `options.sharedModel` out of the `super` call

File: src/linkPanel/linkPanel.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    this._sharedModel = options.sharedModel;
+    this.id = 'glue-link-editor';
```
Comment (trungleduc) [inline src/linkPanel/linkPanel.ts:13]: Do we need `id`?

Comment (brichet) [reply src/linkPanel/linkPanel.ts]: Right, it's copied from the previous PR when it was a main area widget.

File: src/linkPanel/linkPanel.ts (hunk 1 above)
Comment (brichet) [reply src/linkPanel/linkPanel.ts]: fixed

File: src/linkPanel/linkPanel.ts (hunk 2 above)
Comment (brichet) [reply src/linkPanel/linkPanel.ts]: Fixed

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks @brichet

**Pull request merged**